src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const history = [];
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
you didnt have default;
[CODE STYLE]: switch/case should always have default case for error handling.
its from checklist⬆️

Reply:
thanks for the review, fixed it ✅